Unreal/CarlaUE4/Plugins/Carla/Source/Carla/Settings/CarlaSettings.cpp

```diff
@@ -110,6 +118,19 @@ static void LoadSettingsFromConfig(
   ConfigFile.GetInt(S_CARLA_LEVELSETTINGS, TEXT("WeatherId"), Settings.WeatherId);
   ConfigFile.GetInt(S_CARLA_LEVELSETTINGS, TEXT("SeedVehicles"), Settings.SeedVehicles);
   ConfigFile.GetInt(S_CARLA_LEVELSETTINGS, TEXT("SeedPedestrians"), Settings.SeedPedestrians);
+
+  // QualitySettings.
+  FString sDefaultLevel;
+  ConfigFile.GetString(S_CARLA_QUALITYSETTINGS, TEXT("DefaultLevel"), sDefaultLevel);
+  if(!Settings.SetQualitySettingsLevel(FQualitySettings::FromString(sDefaultLevel)))
+  {
+	  //error
+  } else
+  {
+	  //apply pre-restart.... 
```

Comment:
Something to be done here?

Reply:
Uhm...No, nothing to be done yet, it's planned to be added with operations that would need a reset of the level. But there is no reason to do that right now.